docs/docs-sidebar.json

```diff
@@ -2,12 +2,7 @@
   {
     "type": "category",
     "label": "Overview",
-    "items": [
-      "README",
-      "developer-guide/get-started",
-      "whats-new",
-      "upgrade-guide"
-    ]
+    "items": ["README", "developer-guide/get-started", "whats-new", "upgrade-guide"]
```

Comment:
Do we understand why the layout of this file suddenly changed? Modifications further down somehow triggered prettier?

Reply:
Yes, it's changed by prettier after saving other changes.
Do you want to revert it back?